src/server.h

```diff
@@ -415,6 +416,7 @@ typedef enum blocking_type {
     BLOCKED_ZSET,    /* BZPOP et al. */
     BLOCKED_POSTPONE, /* Blocked by processCommand, re-try processing later. */
     BLOCKED_SHUTDOWN, /* SHUTDOWN. */
+    BLOCKED_WAIT_PREREPL, /* WAIT for pre-replication and then run the command. */
```

Comment:
I found the words pre-replication is shown in several places, but I do not understand what it is (sorry missed your previous pr), Can you add a more detail explanations here so everyone can know its meaning after some time later, Thanks

Reply:
will address in the follow up PR